call-apis-on-users-behalf/others-api/langchain-next-js/src/lib/auth0-ai.ts

```diff
@@ -19,8 +19,7 @@ const withAccessTokenForConnection = (connection: string, scopes: string[]) =>
     subjectTokenType: SUBJECT_TOKEN_TYPES.SUBJECT_TYPE_ACCESS_TOKEN,
   });
 
-// Connection for Google services
 export const withGmailSearch = withAccessTokenForConnection(
-  'google-oauth2',
-  ['https://www.googleapis.com/auth/gmail.readonly'],
+  "google-oauth2",
+  ["https://mail.google.com/"]
```

Comment:
I don't think we should be changing the requested scopes here... was there a reason?